### PR TITLE
Y_Update PlaceholderParser.java

### PR DESCRIPTION
**Pull request by abdelmawla56**
i have solved the NestedPlaceholderPart does not recursively resolve placeholders #34020 issue

### CODE REVIEW

**Comment by abdelmawla56** [comment]
hello @vip-delete @spring-projects i have solved the issue please allow me to edit the file

File: spring-core/src/main/java/org/springframework/util/PlaceholderParser.java [hunk 1]
```diff
@@ -443,34 +443,30 @@ record SimplePlaceholderPart(String text, String key, @Nullable String fallback)
 
 		@Override
 		public String resolve(PartResolutionContext resolutionContext) {
-			String resolvedValue = resolveToText(resolutionContext, this.key);
```
Comment (snicoll) [inline spring-core/src/main/java/org/springframework/util/PlaceholderParser.java:446]: Please revert all the formatting change as it makes it harder for us to see what you've changed.

**Review by snicoll — CHANGES_REQUESTED** [review]
Thanks for he PR

> i have solved the issue please allow me to edit the file

I am not sure what you're asking.

Can you please review the note and add a unit test that shows the use case described in the issue? You also need to sign the CLA.

**Comment by snicoll** [comment]
Looking at the code I am not sure what that meant to accomplish. If you want to contribute, please create a proper branch so that you can work in your IDE and add some tests. I've taken over, see #34020

**Pull request closed without merging**